.github/workflows/ci.yaml

```diff
@@ -823,7 +823,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ${{ fromJson(needs.info.outputs.python_versions) }}
-    name: Split tests for full run Python ${{ matrix.python-version }}
+    name: Split tests for full run
```

Comment:
Small unrelated improvement. The matrix values can be added automatically to the job name like it's the case for the `Prepare dependencies` job.
https://github.com/home-assistant/core/blob/96de4b3828c1ec3f17e7573e58a846ef43a6a647/.github/workflows/ci.yaml#L459-L461

The job name will then be:
```diff
-Split tests for full run Python 3.12
+Split tests for full run (3.12)
-Split tests for full run Python 3.13
+Split tests for full run (3.13)
```